README.md

```diff
@@ -1,5 +1,5 @@
 # blog-app
-The Blog app is a classic example of a blog website. I create a fully functional website that shows the list of posts and empower readers to interact with them by adding comments and liking posts.
+The Blog app is a classic example of a blog website. It's a functional website that shows the list of posts and empower readers to interact with them by adding comments and liking posts.
 
 ## Getting Started
  Go ahead and check the most recent version of each of this in the code editor:
```

Comment:
- [ ] In the getting started section, add the complete instructions on how to set up locally. Remember to include all the necessary commands.

Reply:
![God](https://user-images.githubusercontent.com/63972670/146657521-49d60207-5eac-4f03-821d-2f4d92d75725.JPG)

Reply:
Migration work well at my end. the code is working.

Reply:
![ta](https://user-images.githubusercontent.com/63972670/146657612-88d383cc-641c-4686-ae35-bd3dc670ff7c.JPG)

Reply:
![tc](https://user-images.githubusercontent.com/63972670/146657622-db79c9d9-ce59-4ab0-8e2f-c7a5dfa0335c.JPG)

Reply:
Let me look further into this and also set up your repo on another computer. I'll get back to you in a few.

Reply:
okay sir. Thank you

Reply:
You have this migration file `rename_column :posts, :string, :title` and apparently there is no table column of string.
Remember the format shouls be like `rename_column :table, :old_column, :new_column`. For your case, the old column name string does not exist. Check the migration file `FixTitleInPost`

This is the reason for this error.

Reply:
That was exactly the issue. It has been solved. thank you. @mke2111